pkg/apiserver/apiserver.go

```diff
@@ -26,6 +26,9 @@ import (
 	"sync"
 	"time"
 
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/util/retry"
+
```

Comment:
Need to be put together with the following group

Reply:
Code formatted  by `hack/update-gofmt.sh` and `update-goimports.sh`.